src/store/GlobalStore.ts

```diff
@@ -0,0 +1,20 @@
+import {create} from 'zustand';
+import {devtools} from 'zustand/middleware';
+import {shallow} from 'zustand/shallow';
+
+import createTriggersSlice, {TriggersSlice} from './TriggersSlice';
+
+/* Create store. For more information,
+check https://github.com/pmndrs/zustand/blob/main/docs/guides/typescript.md#basic-usage  */
+const useGlobalStore = create<TriggersSlice>()(
```

Comment:
It can't be global store, it has to be local. Otherwise, we will i.e. have problems with running it in Cloud.